apptools/apptools-linux-tests/tests.full.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="./testcase.xsl"?>
+<test_definition>
+  <suite category="Crosswalk App Tools" name="wrt-apptools-linux-tests">
+    <set name="apptools" type="pyunit">
+      <testcase component="Crosswalk App Tools/Project Create" execution_type="auto" id="Crosswalk_create" priority="P1" purpose="Validate if App Tool can work well with create option" status="approved" type="Functional" subcase="3">
```

Comment:
Please update component name with "Crosswalk App Tools/CLI", and add prefix "Linux - "before purpose to distinguish different platforms.